fe/tailwind.config.js

```diff
@@ -39,12 +39,28 @@ module.exports = {
                     "linear-gradient(180deg, #F0F0F0 0%, #F5DED4 100%)"
             },
             spacing: {
+                "8.5": '2.125rem',
                 "8xl": "96rem",
-                "9xl": "128rem"
+                "9xl": "128rem",
+                "23.25" : "5.813rem",
+                "44.5" : "11.156rem",
```

Comment:
Update this as discussed. 
Please add a comment in Figma so I can update it there as well.